cherrypy/test/test_httputil.py

```diff
@@ -79,3 +80,84 @@ def test_invalid_status(status_code, error_msg):
     """Check that invalid status cause certain errors."""
     with pytest.raises(ValueError, match=error_msg):
         httputil.valid_status(status_code)
+
+
+@pytest.mark.parametrize(
+    ('header_content', 'value', 'params'),
+    [
+        ('application/x-www-form-urlencoded',
+         'application/x-www-form-urlencoded', {}),  # default_content_type
```

Comment:
Here's an idea. The params get converted into long strings that are hard to follow, in the pytest output. To combat this, a typical solution would be to wrap each param via `pytest.param(..., id='...')` with that `id` made a human-readable kebab-case identifier for each case that would hint the reader what this parameter's high-level meaning is.
I got an example for you:
```suggestion
        pytest.param(
            'application/x-www-form-urlencoded',
            'application/x-www-form-urlencoded',
            {},
            id='default-content-type',
        ),
```